web/src/components/base/Checkbox.tsx

```diff
@@ -1,39 +1,19 @@
 import styled from '@emotion/styled'
+import { Checkbox as MUICheckbox, FormControlLabel } from '@mui/material'
 import React, { ReactElement } from 'react'
 
-const Container = styled.div`
-  display: flex;
-  flex: 1;
-  gap: 10px;
-`
-
-const StyledCheckbox = styled.input`
-  cursor: pointer;
-  accent-color: ${props => props.theme.colors.themeColor};
-  width: 20px;
-  height: 20px;
-  min-width: 20px;
-  align-self: center;
-`
-
-const StyledLabel = styled.label`
-  color: ${props => props.theme.colors.textColor};
-  padding: 4px;
-  cursor: pointer;
+const StyledLabel = styled(FormControlLabel)`
+  font-weight: 400;
```

Comment:
🤔 Isn't a font-weight of 400 the default?

Reply:
Good point, I'll already use the Typography here :)